addons/stock/wizard/product_replenish.py

```diff
@@ -35,6 +36,11 @@ def _onchange_product_id(self):
         if not self.env.context.get('default_quantity'):
             self.quantity = abs(self.forecasted_quantity) if self.forecasted_quantity < 0 else 1
 
+    @api.depends('product_id', 'product_id.uom_id', 'product_id.uom_ids', 'product_id.seller_ids', 'product_id.seller_ids.product_uom_id')
+    def _compute_allowed_uom_ids(self):
+        for rec in self:
+            rec.allowed_uom_ids = rec.product_id.uom_id | rec.product_id.uom_ids | rec.product_id.seller_ids.product_uom_id
```

Comment:
It could be in a static domain no?

Reply:
Nested fields don't work properly in static domains.